gymnasium/wrappers/__init__.py

```diff
@@ -46,27 +45,135 @@
-from gymnasium.wrappers.time_aware_observation import TimeAwareObservation
-from gymnasium.wrappers.time_limit import TimeLimit
-from gymnasium.wrappers.transform_observation import TransformObservation
-from gymnasium.wrappers.transform_reward import TransformReward
-from gymnasium.wrappers.vector_list_info import VectorListInfo
+from gymnasium.wrappers.common import (
+    Autoreset,
+    OrderEnforcing,
+    PassiveEnvChecker,
+    RecordEpisodeStatistics,
+    TimeLimit,
+)
+from gymnasium.wrappers.rendering import HumanRendering, RecordVideo, RenderCollection
+from gymnasium.wrappers.stateful_action import StickyAction
+from gymnasium.wrappers.stateful_observation import (
+    DelayObservation,
+    FrameStackObservation,
+    MaxAndSkipObservation,
+    NormalizeObservation,
+    TimeAwareObservation,
+)
+from gymnasium.wrappers.stateful_reward import NormalizeReward
+from gymnasium.wrappers.transform_action import (
+    ClipAction,
+    RescaleAction,
+    TransformAction,
+)
+from gymnasium.wrappers.transform_observation import (
+    DtypeObservation,
+    FilterObservation,
+    FlattenObservation,
+    GrayscaleObservation,
+    RenderObservation,
+    RescaleObservation,
+    ReshapeObservation,
+    ResizeObservation,
+    TransformObservation,
+)
+from gymnasium.wrappers.transform_reward import ClipReward, TransformReward
+
+
+__all__ = [
+    "vector",
+    # --- Observation wrappers ---
+    "AtariPreprocessing",
+    "DelayObservation",
+    "DtypeObservation",
+    "FilterObservation",
+    "FlattenObservation",
+    "FrameStackObservation",
+    "GrayscaleObservation",
+    "TransformObservation",
+    "MaxAndSkipObservation",
+    "NormalizeObservation",
+    "RenderObservation",
+    "ResizeObservation",
+    "ReshapeObservation",
+    "RescaleObservation",
+    "TimeAwareObservation",
+    # --- Action Wrappers ---
+    "ClipAction",
+    "TransformAction",
+    "RescaleAction",
+    # "NanAction",
+    "StickyAction",
+    # --- Reward wrappers ---
+    "ClipReward",
+    "TransformReward",
+    "NormalizeReward",
+    # --- Common ---
+    "TimeLimit",
+    "Autoreset",
+    "PassiveEnvChecker",
+    "OrderEnforcing",
+    "RecordEpisodeStatistics",
+    # --- Rendering ---
+    "RenderCollection",
+    "RecordVideo",
+    "HumanRendering",
+    # --- Conversion ---
+    "JaxToNumpy",
+    "JaxToTorch",
+    "NumpyToTorch",
+]
+
+# As these wrappers requires `jax` or `torch`, they are loaded by runtime for users trying to access them
+#   to avoid `import jax` or `import torch` on `import gymnasium`.
+_wrapper_to_class = {
+    # data converters
+    "JaxToNumpy": "jax_to_numpy",
+    "JaxToTorch": "jax_to_torch",
+    "NumpyToTorch": "numpy_to_torch",
+}
+
+_renamed_wrapper = {
+    "AutoResetWrapper": "Autoreset",
+    "FrameStack": "FrameStackObservation",
+    "PixelObservationWrapper": "RenderObservation",
+    "VectorListInfo": "vector.DictInfoToList",
+}
+
+
+def __getattr__(wrapper_name: str):
```

Comment:
This is necessary for the jax / pytorch wrapper otherwise those modules will be loaded on `import gymnasium` if we want to allow users to do `gymnasium.wrappers.JaxToNumpy` or `from gymnasium.wrappers import JaxToNumpy`